src/routers/handlers/stop-screen/stopScreenHandler.ts

```diff
@@ -0,0 +1,76 @@
+import { Request, Response } from "express";
+import { PrefixedUrls, STOP_TYPE, toStopScreenPrefixedUrl } from "../../../constants";
+import { getLocaleInfo, getLocalesService, selectLang } from "../../../utils/localise";
+import { BaseViewData, GenericHandler, ViewModel } from "../generic";
+import { addSearchParams } from "../../../utils/queryParams";
+import { getUrlWithStopType, getUrlWithTransactionIdAndSubmissionId } from "../../../utils/url";
+import { env } from "../../../config";
+
+interface StopScreenHandlerViewData extends BaseViewData {
+    linkUrls?: string[];
+}
```

Comment:
For the Secure screen I need to pass the DSR  email address and telephone number.  I can pass them in this array, even though they are not link urls as such.
- so either I could add extra params to the ViewData
- or this array could be renamed to be more general??

Reply:
I'll rename to `extraData`

Reply:
👍